clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -61,12 +63,14 @@
 #include "llvm/ADT/STLForwardCompat.h"
 #include "llvm/ADT/StringExtras.h"
 #include "llvm/Demangle/Demangle.h"
+#include "llvm/IR/Cheri.h"
 #include "llvm/IR/DerivedTypes.h"
 #include "llvm/MC/MCSectionMachO.h"
 #include "llvm/Support/Error.h"
 #include "llvm/Support/MathExtras.h"
 #include "llvm/Support/raw_ostream.h"
 #include "llvm/TargetParser/Triple.h"
+#include <iostream>
```

Comment:
Please remove

Reply:
[555004a](https://github.com/CHERIoT-Platform/llvm-project/pull/147/commits/555004a699ff358a49e2e9237b96e4b1fbe549e9) removes it.